src/PartitionGraph.f90

```diff
@@ -1703,28 +1720,42 @@ SUBROUTINE SpatialKernighanLin_PartitionGraph(thisGraph,L1,L2)
 !-------------------------------------------------------------------------------
 !> @brief Calculate metrics relevant to the quality of partition
 !> @param thisGraph the partitioned graph to check metrics of
+!> @param maxnsr the maximum number of source regions
```

Comment:
Possibly reading the code incorrectly, but this returns the max **fraction** of source regions, right? If so, we should let this comment reflect that.